lib/rspec/expectations/configuration.rb

```diff
@@ -56,6 +56,19 @@ def syntax=(values)
         end
       end
 
+      # Configures the maximum character length that RSpec will print while
+      # formatting an object
+      # @param [Fixnum] length the number of characters to limit the formatted output to
+      # @example
+      #   RSpec.configure do |rspec|
+      #     rspec.expect_with :rspec do |c|
+      #       c.max_formatted_output_length = 200
+      #     end
+      #   end
```

Comment:
I think the docs should mention that you can set the length to `nil` to prevent RSpec from doing any truncation.

Reply:
Good idea @myronmarston. https://github.com/rspec/rspec-expectations/pull/1057